Form/Type/MediaType.php

```diff
@@ -77,14 +76,34 @@ public function buildView(FormView $view, FormInterface $form, array $options)
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated To be removed when bumping to Symfony 2.7+, use MediaType::configureOptions() instead
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
+        if ($resolver instanceof OptionsResolver) {
```

Comment:
I know that @Soullivaneuh, but this is the issue here: ```Symfony\Component\Form\FormTypeInterface::setDefaultOptions()``` allows you to use any implementation of ```Symfony\Component\OptionsResolver\OptionsResolverInterface```. The constraint is only narrowed from ```configureOptions()```.